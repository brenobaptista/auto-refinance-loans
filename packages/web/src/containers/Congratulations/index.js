import React from "react";
import styles from "./index.module.css";

const Congratulations = () => {
  return (
    <div className={styles.congratulations}>
      <div className={styles.wrapper}>
        <h1 className={styles.title}>
          <svg
            xmlns="http://www.w3.org/2000/svg"
            width="24"
            height="24"
            fill="none"
            viewBox="0 0 24 24"
            stroke="currentColor"
            className={styles.stars}
          >
            <path
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="2"
              d="M5 3v4M3 5h4M6 17v4m-2-2h4m5-16l2.286 6.857L21 12l-5.714 2.143L13 21l-2.286-6.857L5 12l5.714-2.143L13 3z"
            />
          </svg>
          <span>Congratulations!</span>
        </h1>
        <p className={styles.explanation}>
          You're about to lower your monthly auto loan payment! A loan officer
          will reach out to you shortly to discuss the final steps and confirm
          the loan offer.
        </p>
      </div>
    </div>
  );
};

export default Congratulations;
