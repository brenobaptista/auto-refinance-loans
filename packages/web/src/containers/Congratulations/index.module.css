.congratulations {
  display: flex;
  justify-content: center;
  align-items: center;
  height: 100vh;
  background-color: #000000;
  color: #ffffff;
  padding: 0 24px;
}

@keyframes roll-right {
  0% {
    opacity: 0;
    transform: translate(-16px);
  }

  100% {
    opacity: 1;
    transform: translate(0);
  }
}

.wrapper {
  max-width: 400px;
  animation: roll-right 1s;
}

@keyframes shine {
  0%,
  100% {
    opacity: 1;
    transform: scale(1.2);
  }

  50% {
    opacity: 0.5;
    transform: scale(1);
  }
}

.stars {
  animation: shine 3s infinite;
}

.title {
  display: flex;
  gap: 10px;
  font-size: 1.5rem;
  margin-top: 0;
  padding-top: 32px;
}

.explanation {
  color: #6a6a6a;
  line-height: 1.4;
}
