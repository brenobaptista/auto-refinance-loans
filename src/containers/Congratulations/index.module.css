.congratulations {
  display: flex;
  justify-content: center;
  align-items: center;
  height: 100vh;
  background-color: #000000;
  color: #ffffff;
  padding: 0 24px;
}

.wrapper {
  max-width: 400px;
}

.title {
  display: flex;
  gap: 10px;
  font-size: 1.5rem;
  margin-top: 0;
  padding-top: 32px;
}

.explanation {
  color: #6a6a6a;
  line-height: 1.4;
}
