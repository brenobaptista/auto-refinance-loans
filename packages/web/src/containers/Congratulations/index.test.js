import React from "react";
import { render } from "@testing-library/react";
import Congratulations from "./index";

test("matches snapshot", () => {
  const { asFragment } = render(<Congratulations />);

  expect(asFragment()).toMatchSnapshot();
});
