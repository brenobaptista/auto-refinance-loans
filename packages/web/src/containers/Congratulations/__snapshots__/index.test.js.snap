// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="congratulations"
  >
    <div
      class="wrapper"
    >
      <h1
        class="title"
      >
        <svg
          class="stars"
          fill="none"
          height="24"
          stroke="currentColor"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M5 3v4M3 5h4M6 17v4m-2-2h4m5-16l2.286 6.857L21 12l-5.714 2.143L13 21l-2.286-6.857L5 12l5.714-2.143L13 3z"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
          />
        </svg>
        <span>
          Congratulations!
        </span>
      </h1>
      <p
        class="explanation"
      >
        You're about to lower your monthly auto loan payment! A loan officer will reach out to you shortly to discuss the final steps and confirm the loan offer.
      </p>
    </div>
  </div>
</DocumentFragment>
`;
